### PR TITLE
[SMP-1053]: Increased manager memory request to 2x heap size

### PR DESCRIPTION
**Pull request by avaakash**

### CODE REVIEW

**Comment by arya-harness** [comment]
why only cg-manager, all java based services needs to have pod memory 2x the java memory

**Comment by arya-harness** [comment]
can we also do it as service level too, at service level you can configure it to be at override-demo configuration

**Comment by avaakash** [comment]
Increasing the requests to 2x for all services is going to increase the minimum resources required for harness by around 90%. Should we do it for all or do it for manager services which handles the majority of the requests? I can see tickets with the manager crashing mostly.

**Comment by avaakash** [comment]
Adding at the service level as in adding the default resource constraints in the service deployment charts directly (defaults to override-demo values)?

File: src/harness/override-demo.yaml [hunk 1]
```diff
@@ -478,10 +478,10 @@ platform:
     replicaCount: 1
     resources:
       limits:
-        memory: 3000Mi
```
Comment (akshay-khole) [inline src/harness/override-demo.yaml:481]: do not change demo

**Pull request closed without merging**